ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -477,6 +516,71 @@ private void OnViewChanging(object sender, ScrollViewerViewChangingEventArgs arg
                         }));
         }
 
+        private void OnLoaded(object sender, RoutedEventArgs e)
+        {
+            var scrollViewer = (ScrollViewer)sender;
+            _scrollViewerData[scrollViewer].IsControlLoaded = true;
+
+            UpdateDisableKeyboardBasedScrolling(scrollViewer, _scrollViewerData[scrollViewer].DisableArrowNavigation);
+        }
+
+        private void UpdateDisableKeyboardBasedScrolling(ScrollViewer view, bool disable)
+        {
+            ScrollContentPresenter contentPresenter = _scrollViewerData[view].ContentPresenter;
+            if (disable)
+            {
+                if (contentPresenter == null)
+                {
+                    contentPresenter = FindChild<ScrollContentPresenter>(view);
+                    if (contentPresenter == null)
+                    {
+                        throw new InvalidOperationException("ScrollViewer doesn't seem to contain a ScrollContentPresenter");
+                    }
+                    _scrollViewerData[view].ContentPresenter = contentPresenter;
+                }
+
+                contentPresenter.KeyDown += OnPresenterKeyDown;
```

Comment:
>contentPresenter [](start = 16, length = 16)

Why do we have to listen to the event on the ScrollContentPresenter rather than the ScrollViewer?

Reply:
ScrollViewer processes those events, so they don't end up for "outside" consumption on the KeyDown/Up hook points. 
That's the usual pattern. If a control processes some events, they stop bubbling starting from that control.
This processing happens inside OnKeyDown/Up, but ScrollViewer is sealed so we can't override that implementation.

Reply:
ScrollViewer processes those events, so they don't end up for "outside" consumption on the KeyDown/Up hook points. 
That's the usual pattern. If a control processes some events, they stop bubbling starting from that control.
This processing happens inside OnKeyDown/Up, but ScrollViewer is sealed so we can't override that implementation.

---
In reply to: [156584239](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156584239) [](ancestors = 156584239)

Reply:
ScrollViewer processes those events, so they don't end up for "outside" consumption on the KeyDown/Up hook points. 
That's the usual pattern. If a control processes some events, they stop bubbling starting from that control.
This processing happens inside OnKeyDown/Up, but ScrollViewer is sealed so we can't override that implementation.

---
In reply to: [156584239](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156584239) [](ancestors = 156584239)